src/Microsoft.Azure.WebJobs.ServiceBus/ServiceBusAccountAttribute.cs

```diff
@@ -33,11 +33,13 @@ public ServiceBusAccountAttribute(string account)
         }
 
         /// <summary>
-        /// Gets the name of the ServiceBus connection string to use.
+        /// The name of the app setting containing the Service Bus connection string.
```

Comment:
This would fail in script but we don't have the rule (SA1623) enabled here to help. For properties, we should maintain the gets/sets convention.